src/otx/recipe/anomaly/anomaly_classification/stfpm.yaml

```diff
@@ -0,0 +1,89 @@
+model:
+  class_path: otx.algo.anomaly.stfpm.STFPM
+  init_args:
+    input_size:
+      - 256
+      - 256
+    layers: ["layer1", "layer2", "layer3"]
+    backbone: "resnet18"
+
+optimizer:
+  class_path: torch.optim.SGD
+  init_args:
+    lr: 0.4
+    momentum: 0.9
+    dampening: 0
+    weight_decay: 0.001
+
+engine:
+  task: ANOMALY_CLASSIFICATION
+  device: auto
+
+callback_monitor: train_loss_epoch # val loss is not available as there is no validation set from default dataloader
+
+data: ../../_base_/data/torchvision_base.yaml
+overrides:
+  max_epochs: 100
+  limit_val_batches: 0 # this is set to 0 as the default dataloader does not have validation set. But this also means that the model will not give correct performance numbers
+  callbacks:
+    - class_path: lightning.pytorch.callbacks.EarlyStopping
+      init_args:
+        patience: 5
+  data:
+    task: ANOMALY_CLASSIFICATION
+    config:
+      data_format: mvtec
+      vpm_config:
+        use_bbox: True
+        use_point: False
```

Comment:
@harimkang, do you know what these params are?

Reply:
As far as I know, this is only used in the visual prompting model and does not need to be declared in anomaly.